Bump CKE version to 1.17.11 (#1079)
cybozu-go/neco: docs/artifacts.md

Artifacts
=========

[Artifacts](../artifacts.go) is the collection of components tested in [dctest](../dctest/), which consists of container images, debian package, and CoreOS image.

There are two artifacts files.  When generating neco binaries and the neco deb package, `artifacts.go` is used

> in default.  A CI job uses `artifacts_release.go` by specifying `TAGS=release`

Reviewer: ```suggestion
by default.  `artifacts_release.go` is used only when `release` build tag is given.
```

> for `make` command after checking-out the `release` branch.

Reviewer: ```suggestion
```

    This file describes artifacts to be tested in development.
    It is normally updated with `generate-artifacts` command.

>     **PLEASE DO NOT EDIT MANUALLY** if you are trying to update components in staging or production environment.

Reviewer: I cannot understand this sentence.

Reviewer: For `staging` and `production` environments, `artifacts.go` is _not used_.
Instead, `artifacts_release.go` in the release branch is used.
